docs/sources/architecture/thing-definition.md

```diff
@@ -154,4 +154,50 @@ The channel group type is defined on the same level as the thing types and chann
 </channel-group-type> 
 ```
 
-When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
\ No newline at end of file
+When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
+
+### Default Thing Properties
+The configuration descriptions explained in full detail [here](../architecture/configuration.md) are used in order to provide configuration properties for bindings. These properties are enhanced through configuration descriptions so that the property value can be interpreted in a meaningful manner. However in order to keep the XML based configuration clean there are predefined default thing properties available. The type of these properties can always be handled as a `java.lang.String` so that no further configuration description is necessary. The keys for these properties are defined in the `org.eclipse.smarthome.core.thing.DefaultPropertyKey` enumeration which provides the following property keys:
+
+- Vendor
+- Model
+- Serial Number
+- Hardware Version
+- Firmware Version
+
+To use these properties the operation `getStaticProperties` of the `BaseThingHandlerFactory` class can be overwritten in the concrete thing handler factory class. The following code snippet shows how these thing properties are injected into the thing configuration. 
```

Comment:
This method is unrelated to the default property keys, so this part should imho be removed from this paragraph.